docs/v1/references.bib

```diff
@@ -191,6 +191,11 @@ @ONLINE { ProtoOneOfBackwardsCompatibility
     url = "https://developers.google.com/protocol-buffers/docs/proto3#backwards-compatibility-issues"
 }
 
+@ONLINE { P4Annotations,
+    title = "P4 Annotations",
+    url = "https://p4.org/p4-spec/docs/P4-16-v1.2.0.html#sec-annotations"
```

Comment:
Feel free to change that link so that it refers to the "latest" (working draft) version of the P4 spec, since v1.2.0 does not include anything about structured annotations. This way when your p4-spec PR is merged, at least the link will point to some text that talks about structured annotations.

Reply:
@antoninbas Good suggestion. Would I link to the .mdk form of the spec in GitHub, e.g. https://github.com/p4lang/p4-spec/blob/master/p4-16/spec/P4-16-spec.mdk instead?

Reply:
@antoninbas Regarding the "empty expressionList" topic: I agree it makes more sense, and in my original example I did just that (empty oneof). However @stefanheule commented that it ought to be an empty ExpressionList to be consistent with the P4-16 Spec, namely this cluase:

Reply:
That's fair, I take the blame for the change. But I do think we want the spec and this to be consistent, but if everyone feels like we want to not disambiguate the two, we should also not disambiguate in the spec.  Or we disambiguate the same way in both places.

Reply:
@chrispsommers you can use this link: https://p4.org/p4-spec/docs/P4-16-working-spec.html#sec-annotations

let's use the other thread to discuss empty annotations